### DISCUSSION TITLE
Switch Inverse Color

### DISCUSSION THREAD

**Discussion by annefranca** [Telefonica/mistica-design#1747] [Proposals]
### Goal

Avaliar a proposta da implementação do Theme Context Inverse no componente Switch.

### Context

Na jornada de Casa Inteligente, precisamos aplicar o Switch no backgroundBrandContainerAlternative (token extended) respeitando os tokens de cor que foram aprovados no conceito do Novo App.

### Impact

Ao adicionar o Theme Context Inverse no componente Switch, estaríamos padronizando uma definição já existente em outros componentes da biblioteca Mística.

### Discussion Questions

- Vocês têm algum estudo em relação ao Theme Inverse para o componente Switch?
- Vocês têm alguma outro direcionamento de como poderíamos chegar no resultado esperado?

### Do you have a visual proposal?

https://www.figma.com/file/NzeGI73CNQUaaaQYXn6UIt/Switch-(inverse)?type=design&node-id=1%3A86&mode=design&t=1wNwyO1vs303FNc1-1

**Comment by yceballost** [comment]
We need this, absolutely

That's a very good material to help us to define inverse controls. We have pending this project for a long time https://github.com/Telefonica/mistica-design/issues/736

@aweell was working a proposal for this (and the rest of the controls) here (you can also find this figma in the issue) 
https://www.figma.com/file/1F6jqwc0XCvNO8LWxTnDea/Untitled?type=design&node-id=125-8521&mode=design&t=iMhdK6SHTmCbcASJ-0
Alex, maybe you need to change the name of this file to avoid "Untitled" 😁

I will link this discussion to this issue

I recommend you to create a ticket in your Jira board to priorize it!

**Reply by aweell** [reply]
> Alex, maybe you need to change the name of this file to avoid "Untitled"

Indeed, renamed to "inverse controls POC"

**Reply by annefranca** [reply]
Ticket criado https://jira.tid.es/browse/OBVIVO-2597

**Comment by aweell** [comment]
This will be included in #736
